references/gcp-sa-auth-shared-flow/test/apiproxy/proxies/default.xml

```diff
@@ -0,0 +1,46 @@
+<!--
+ Copyright 2020 Google LLC
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+<ProxyEndpoint name="default">
+    <PreFlow name="PreFlow">
+        <Request>
+            <Step>
+                <Name>KV.LookupSAKey</Name>
+            </Step>
+            <Step>
+                <Name>AM.SetScope</Name>
+            </Step>
+            <Step>
+                <Name>FC.AccessToken</Name>
+            </Step>
+            <Step>
+                <Name>AM.SetAuthHeader</Name>
+            </Step>
+        </Request>
+        <Response/>
+    </PreFlow>
+    <Flows/>
+    <PostFlow name="PostFlow">
+        <Request/>
+        <Response/>
+    </PostFlow>
+    <HTTPProxyConnection>
+        <BasePath>/token-validation/v0</BasePath>
```

Comment:
interested in why we chose v0 and not v1?

Reply:
Intention behind was: because this is just a proxy to test the SF and not production-ready implementation.